src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/svn/SVNStatusRenderer.java

```diff
@@ -92,6 +92,7 @@ private String descForStatus(String str)
       
       switch (c)
       {
+         // TODO Similar to GitStatusRender unsure about cases
```

Comment:
Same answer as for GitStatusRenderer. Localize the returns strings, but not the case values. These strings show up as tooltips in the Subversion UI.

I'd show a screenshot but then I'd have to remember how to setup svn.